src/pages/prediction/leaderboard.tsx

```diff
@@ -1,3 +1,18 @@
+import LocalReduxProvider from 'contexts/LocalRedux/Provider'
+import makeStore from 'contexts/LocalRedux/makeStore'
+import reducers, { initialState } from 'state/predictions'
+import ConfigProvider, { config } from '../../views/Predictions/context/ConfigProvider'
+
 import PredictionsLeaderboard from '../../views/Predictions/Leaderboard'
 
-export default PredictionsLeaderboard
+const store = makeStore(reducers, initialState, config)
+
+export default () => {
+  return (
+    <ConfigProvider>
+      <LocalReduxProvider store={store}>
```

Comment:
we don't need to make different store for prediction and leaderboard page, right?

Reply:
Good catch.